metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -220,6 +221,16 @@ object UserConfiguration {
           |See https://scalameta.org/metals/docs/contributors/remote-language-server.html for
           |documentation on remote language servers.
           |""".stripMargin
+      ),
+      UserConfigurationOption(
+        "fallback-scala-version",
```

Comment:
We should also create a PR to metals-vscode and add the new configuration option to `package.json`, we should also only allow to select from a number of values (I think that should be supported)

Reply:
Yep, I have a branch with these changes. Will publish them soon.
Decided to delay this PR creation because I expected there might be some changes in option name and it's description)